### PR TITLE
fix(switch-button): add night mode style

### PR DESCRIPTION
**Pull request by djcouto**
##### Default
Screenshot 2022-03-04 at 09 54 26

##### Disabled
Screenshot 2022-03-04 at 18 06 28

### CODE REVIEW

File: packages/ui-components/src/components/switch-button/switch-button.scss [hunk 1]
```diff
@@ -1,11 +1,39 @@
...
-$sqr-radius: 2px;
-$icon-size: 16px;
+$btn-height: 20px;
+$btn-width: 40px;
+$sqr-size: 16px;
+$sqr-padding-x: 2px;
+$icon-size: 12px;
+
+@function get-sqr-x-translation() {
+	@return $btn-width - $sqr-size - $sqr-padding-x * 2;
```
Comment (ricardogoncalves89) [inline packages/ui-components/src/components/switch-button/switch-button.scss:10]: If I set the `--btn-height` var the calculation don't return the correct value. 

Suggestion: 
```
:host {
	--btn-height: 20px;

	$btn-height: var(--btn-height);
}
```
after that use only the `$btn-height` for styling and calculations

**Comment by pedrorebelocosta** [comment]
This is pending as it requires https://github.com/kelvininc/ui-components/pull/35 and https://github.com/kelvininc/ui-components/pull/36 to be merged right?

**Review by ricardogoncalves89 — CHANGES_REQUESTED** [review]

File: packages/ui-components/src/components/switch-button/switch-button.scss (hunk 1 above)
Comment (djcouto) [reply packages/ui-components/src/components/switch-button/switch-button.scss:10]: You're absolutely correct.

Fixing on the upcoming commit.

**Review by ricardogoncalves89 — APPROVED** [review]
great work with size styling 👌

**Review by pedrorebelocosta — APPROVED** [review]

**Comment by pedrorebelocosta** [comment]
top notch work :D

**Pull request merged**